provision/opentofu/modules/rosa/hcp/variables.tf

```diff
@@ -49,9 +49,8 @@ variable "subnet_cidr_prefix" {
 }
 
 variable "openshift_version" {
-  type     = string
-  default  = "4.15.5"
-  nullable = false
```

Comment:
@kami619 I think we need to keep `nullable = false`. Without it an empty value is possible and the default value won't be applied.